schema/index.js

```diff
@@ -27,8 +29,10 @@ module.exports = db => {
   Place.establishment = Place.belongsTo(Establishment, { as: 'establishment' });
 
   Role.profile = Role.belongsTo(Profile);
+  TrainingModule.profile = TrainingModule.belongsTo(Profile);
```

Comment:
I don't _think_ we need this, as the data flow will only ever be from a `Profile` to a`TrainingModule` - i.e. we'll never have an arbitrary module and need to know the profile it's attached to.

Reply:
OK